src/app/compose/draftdesk.service.ts

```diff
@@ -57,7 +57,6 @@ export class DraftFormModel {
 
     public static create(draftId: number, fromAddress: FromAddress, to: string, subject: string, preview?: string): DraftFormModel {
         const ret = new DraftFormModel();
-        ret.from = fromAddress.email;
```

Comment:
Hm, I meant move the "pick from address" code to here, not remove it. This is the model, so I'd suggest do more work in the model, and less in the view

Reply:
I understood the issue. My thought was do what you're doing, but do it in draftdesk.service instead, where its closer to the action, instead of the view

Reply:
Everything should be fixed now. Tadzik and I looked through it and just liked he described above - we sorted Froms, making the main identity win.